pandas/tests/io/formats/test_format.py

```diff
@@ -2910,6 +2910,15 @@ def test_format(self):
         assert result[0] == " 12.0"
         assert result[1] == "  0.0"
 
+    def test_output_display_precision_trailing_zeroes(self):
```

Comment:
can you also test with the 2 as display.precision (ideally parameterize), but ok to just list a 2nd case if easier

Reply:
@jreback i actually didn’t have time to address the second test but I saw that you merged the PR ald. I assume that means the test is fine. Let me know otherwise.